tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/distribution/SimpleSymmetricClusterTest.java

```diff
@@ -127,6 +136,80 @@ public void testSimple() throws Exception {
 
    }
 
+   @Test
+   public void testWildcardFlowControl() throws Exception {
```

Comment:
@jbertram how long this test would take to run? Perhaps we should have this one test under SoakTests? (No need to make a RealServerTest.. just put it under soak *if * this is a long running test?

Reply:
The test is actually quite fast so I don't think it's necessary to move. It finishes in around 4 seconds on my laptop.